[18-May-21 20:15:16] [Scholapp server] [Process: 16956] [Scholapp] - INFO - Setting up flask_microservices.video_microservice.video_app
[18-May-21 20:15:16] [Scholapp server] [Process: 16956] [Scholapp] - INFO - Setting up was successful
[21-May-21 12:42:01] [Scholapp server] [Process: 17780] [Scholapp] - INFO - Setting up flask_microservices.test_microservice.test_app
[21-May-21 12:42:02] [Scholapp server] [Process: 17780] [Scholapp] - INFO - Deserializing tests
[21-May-21 12:42:02] [Scholapp server] [Process: 17780] [Scholapp] - INFO - Setting up was successful
[21-May-21 12:42:11] [Scholapp server] [Process: 17780] [Scholapp] - INFO - HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /Login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BAAC8F34F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
[21-May-21 12:42:13] [Scholapp server] [Process: 17780] [Scholapp] - INFO - HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /Login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BAAC8F33D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
[21-May-21 12:42:25] [Scholapp server] [Process: 17780] [Scholapp] - INFO - HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /Login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BAAC91B580>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
[21-May-21 12:42:27] [Scholapp server] [Process: 17780] [Scholapp] - INFO - HTTPConnectionPool(host='127.0.0.1', port=5001): Max retries exceeded with url: /Login (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BAAC9294C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))
[21-May-21 12:43:12] [Scholapp server] [Process: 22608] [Scholapp] - INFO - Setting up flask_microservices.test_microservice.test_app
[21-May-21 12:43:12] [Scholapp server] [Process: 22608] [Scholapp] - INFO - Deserializing tests
[21-May-21 12:43:12] [Scholapp server] [Process: 22608] [Scholapp] - INFO - Setting up was successful
